en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -0,0 +1,98 @@
+-   Capability of viewing and configuring logs without accessing the file system.
+-   User management with [external user stores](../../setup/install_and_setup_overview/#data-stores).
+-   [Readiness probe](../../setup/deployment/health_check) for health checking.
+-   Enhanced experience for [handling security](../../setup/security/encrypting_plain_text) (Ciphertool, Securevault, Kubernetes Secrets).
+-   [Hot deployment](../../develop/hot-deployment) capability.
+-   Schema-based XML to JSON transformations with the [JSON Transform mediator](../../references/mediators/json-Transform-Mediator).
+-   [Swagger generation for data services](../../develop/advanced-development/using-swagger-for-apis).
+-   [Transaction counter](../../setup/deployment/deployment_checklist/#monitoring-transaction-counts) for services.
+-   [Environment/System variable support](../../setup/dynamic_server_configurations) for Kubernetes environments.
+-   [Enhanced Micro Integrator CLI](../../administer-and-observe/using-the-command-line-interface).
+
+### New in WSO2 Integration Studio**
+
+The following features and improvements are available for <b>WSO2 Integration Studio</b> EI 7.1.0:
+
+Revamped [<b>Connectors</b>](../../references/connectors/connectors-overview) functionality with improved user experience:
+
+-   Redesigned connector configuration view.
+-   New externalizable connection configuration view for connectors.
+
+Improved capabilities to support CI/CD with WSO2 Micro Integrator:
+
+!!! Note
+    For more information, read about the [<b>integration project</b>](../../develop/create-integration-project) and the [<b>Docker project</b>](../../develop/create-docker-project).
+
+-   Capability to create the integration solution as a **Maven Multi Module** project.
+-   Embedded maven support to easily build and push projects.
+-   Maven profiles allowing you to build individual integration modules.
+-   Capability to import/export maven project (integration project).
+-   Capability to import/export projects with/without copying to the workspace.
+-   Capability to import existing modules into maven multi module project (integration project).
+-   Capability to build and push Docker exporter using maven.
+-   Capability to push images to a remote repository via the Docker exporter.
+-   Capability to encrypt secrets in Docker/Kubernetes modules of the integration project by using the **Cipher Tool**.
+
+Improvements to the [<b>embedded Micro Integrator (MI)</b>](../../develop/using-embedded-micro-integrator):
+
+-	WSO2 MI 1.2.0 as the embedded server runtime.
+-	Hot deployment capability for the embedded MI.
+-   Capability to configure the embedded Micro Integrator via the tool.
+-   Capability to Run and Debug synapse configurations with MI 1.2.0.
+-   Capability to view deployed services using the **Runtime Services** view.
+-   Capability to view and monitor deployed artifacts using the **embedded** WSO2 EI Monitoring Dashboard.
```

Comment:
can we take the word Capability up and say something like , brings you following Capabilities and then list them. Repetition of it doesn't sound nice